File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelContainedInSpatialStructure/IfcRelContainedInSpatialStructure.cs
public class IfcRelContainedInSpatialStructure : IfcRelConnects
{
    public SET [1:?] OF IfcProduct RelatedElements { get; set; }
    public IfcSpatialElement RelatingStructure { get; set; }

    // === WHERE CLAUSES ===
    // WR31 : SIZEOF(QUERY(temp <* RelatedElements | 'IFC4X3_DEV_73740fe4.IFCSPATIALSTRUCTUREELEMENT' IN TYPEOF(temp))) = 0
}

File: src/main/resources/output/IFC4X3_js/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelContainedInSpatialStructure/IfcRelContainedInSpatialStructure.js
class IfcRelContainedInSpatialStructure extends IfcRelConnects {
    constructor() {
        /** @type {SET [1:?] OF IFCPRODUCT} */
        this.RelatedElements = null;
        /** @type {IFCSPATIALELEMENT} */
        this.RelatingStructure = null;
    }

    // === WHERE CLAUSES ===
    // WR31 : SIZEOF(QUERY(temp <* RelatedElements | 'IFC4X3_DEV_73740fe4.IFCSPATIALSTRUCTUREELEMENT' IN TYPEOF(temp))) = 0
}
